modules/docker-worker/main.tf

```diff
@@ -134,6 +134,7 @@ EOF
   provisioner "remote-exec" {
     inline = [
       "start docker-worker",
+      "echo performance | sudo tee /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor >/dev/null",
```

Comment:
Is the sudo required here, i.e. is this script running as root by default?

Reply:
Good point. Notice I added this to the [docker-worker startup script](https://github.com/taskcluster/docker-worker/commit/5479b900298657f7d2956cceec56b8e3f6cbe9a7) for the case the machine is rebooted.